modules/services/pass-secret-service.nix

```diff
@@ -16,33 +15,48 @@ in {
     package = mkPackageOption pkgs "pass-secret-service" { };
 
     storePath = mkOption {
-      type = with types; nullOr str;
-      default = null;
-      defaultText = "~/.password-store";
+      type = types.str;
+      default = "";
+      defaultText = "$HOME/.password-store";
       example = "/home/user/.local/share/password-store";
-      description = "Absolute path to password store.";
+      description = ''
+        Absolute path to password store.
+      '';
     };
   };
 
   config = mkIf cfg.enable {
     assertions = [
       (hm.assertions.assertPlatform "services.pass-secret-service" pkgs
         platforms.linux)
+      {
+        assertion = !config.services.gnome-keyring.enable;
+        message = "Only one secrets service per user can be enabled at a time";
```

Comment:
Maybe mention the conflicting modules

Reply:
Yeah, I forgot to. I went and double-checked there weren't other services with modules supporting `org.freedesktop.secrets`, but forgot to go back and list them. I'll do so.

Reply:
Added in b1bfb4dd